components/Button/src/index.css

```diff
@@ -96,6 +96,21 @@
   padding-inline-start: var(--denhaag-button-large-size-padding-inline, var(--denhaag-button-padding-inline));
 }
 
+.denhaag-button--outline-dark-action {
+  border-style: solid;
+  background-color: var(--denhaag-button-outline-dark-action-background-color);
+  border-color: var(--denhaag-button-outline-dark-action-border-color);
+  border-width: var(--denhaag-button-border-width);
+  color: var(--denhaag-button-outline-dark-action-color);
+}
+
+.denhaag-button--outline-dark-action.denhaag-button--hover,
+.denhaag-button--outline-dark-action:hover {
+  background-color: var(--denhaag-button-secondary-action-hover-background-color);
+  border-color: var(--denhaag-button-secondary-action-hover-border-color);
+  color: var(--denhaag-button-secondary-action-hover-color);
+}
```

Comment:
I'm struggling with this BEM class. `--outline-dark-action` is a visually descriptive name, which I think we should avoid. The style is more of a tertiary action.

Reply:
@bddjong and move the styles to the button component? tertiary/outline action variant is not for global usage, that is why I put it here.